### PR TITLE
fix(chat): Error messages appear for Admin are impossible to read (Issue #1944, #1959)

### PR DESCRIPTION
**Pull request by denys-kolomiitsev**
**Description:**

- Fixed toast width for Mobile view and toast line break.

Issues:

- Issue #1944 
- Issue #1959 

**UI changes**

image

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by denys-kolomiitsev** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1970.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by denys-kolomiitsev** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1970.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by denys-kolomiitsev** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1970.nightly-test.deltixhub.io
>E2E tests status: success

File: apps/chat/src/components/Toasts/Toasts.tsx [hunk 1]
```diff
@@ -66,10 +68,16 @@ export const Toasts = () => (
                   toast.icon
                 )}
               </span>
-              <div className="mx-0.5 whitespace-pre-wrap text-sm leading-[21px] text-primary *:!whitespace-pre-wrap">
+              <div
+                style={{ wordBreak: 'break-word' }}
```
Comment (IlyaBondar) [inline apps/chat/src/components/Toasts/Toasts.tsx:72]: @denys-kolomiitsev , did you try `break-words` class instead?

**Comment by denys-kolomiitsev** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1970.nightly-test.deltixhub.io
>E2E tests status: success

**Review by IlyaBondar — APPROVED** [review]

File: apps/chat/src/components/Toasts/Toasts.tsx (hunk 1 above)
Comment (denys-kolomiitsev) [reply apps/chat/src/components/Toasts/Toasts.tsx:72]: Unfortunately, this class is not working here as needed.

**Comment by denys-kolomiitsev** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1970.nightly-test.deltixhub.io
>E2E tests status: success

**Pull request merged**